js/src/redux/providers/balancesActions.js

```diff
@@ -65,12 +65,12 @@ function setBalances (_balances) {
           const oldValue = nextTokens[tokenIndex].value;
 
           // If received a token/eth (old value < new value), notify
-          if (oldValue.lt(value) && accounts[address]) {
+          if (oldValue.lt(value) && accounts[address] && !skipNotifications) {
             const account = accounts[address];
             const txValue = value.minus(oldValue);
 
             const redirectToAccount = () => {
-              const route = `/account/${account.address}`;
+              const route = `/accounts/${account.address}`;
```

Comment:
Picked this up last night as well. I guess we actually need to externalise the route definitions somehow so we don't miss these again in the future.